README.md

```diff
@@ -1,85 +1,150 @@
-  * MetricsServerAddon: adds metrics server (pre-req for HPA and other monitoring tools)
-  * NginxAddon: installs NGINX ingress controller 
-* Onboard one or many teams into designated clusters. Examples are provided under `lib/teams`, however due to uniqueness of every team, clients are expected to supply implementation of the `TeamSetup` interface.
-
-Note: currently tested version of CDK with the blueprint is 1.76 (affects bootstrapping as well, see below). It will be migrated shortly to 1.84 (or later). 
-## Bootstrapping
-Each combination of target account and region must be bootstrapped prior to deploying stacks.
-Bootstrapping is an process of creating IAM roles and lambda functions that can execute some of the common CDK constructs.
+The repository contains the source code and configuration for the EKS Shared Services Platform reference architecture. 
 
-Example: 
-```   
-  cdk bootstrap aws://<AWS_ACCOUNT_ID>/us-east-1
+## Getting Started 
+
+### Install CDK 
+
+This reference architecture leverages [AWS Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/). Install CDK via the following.
+
+```bash
+npm install -g aws-cdk
 ```
-In addition to the regular [environment bootstrapping](https://docs.aws.amazon.com/cdk/latest/guide/bootstrapping.html) pipeline bootstrapping for pipelines requires a new style of bootstrapping. Execute (with account admin privileges) the command in bootstrap-pipeline.sh.  
 
-## cdk-eks-blueprint
-Supports context variables (specify in cdk.json, cdk.context.json or pass with -c command line option):
+Verify the installation.
 
-- instanceType: (defaulted to "t3.medium") Type of instance for the EKS cluster, must be a valid instance type like t3.medium
-- vpc: Specifies whether to use an existing VPC (if specified) or create a new one if not specified.
-- minSize: Min cluster size, must be positive integer greater than 0 (default 1).
-- maxSize: Max cluster size, must be greater than minSize.
-- vpcSubnets ="List of VPC subnets for cluster provisioning (unsupported yet)
+```bash
+cdk --version
+```
 
-## Usage
+### Project setup
 
-Create a new CDK project and add the following to the dependency section of the package.json file:
+Create a new CDK project. 
+
+```bash
+cdk init app --language typescript
 ```
-    "@shapirov/cdk-eks-blueprint": "0.1.4"
+
+Bootstrap your environment. For more information see Boostrapping below.  
+
+```bash
+cdk bootstrap aws://<AWS_ACCOUNT_ID>/<AWS_REGION>
 ```
-Note: the module name will be migrated to aws in the near future. 
 
-In the main file (or anywhere in your classes) import CDK EKS blueprint and configure it with various modules:
+### Usage
 
-```ts
+Add this project as a dependency to your CDK project. 
+
+```json
+"dependencies": {
+  "@shapirov/cdk-eks-blueprint": "0.1.4"
+}
+```
+
+Replace the contents of `bin/main.ts` with the following.
```

Comment:
This won't be main.ts in general case. By default CDK will create a file with the same name as the directory where you ran init.

Reply:
I fixed it.